src/App.tsx

```diff
@@ -37,7 +37,7 @@ function App() {
 
 	return (
 		<>
-			<ToastContainer />
+			<ToastContainer style={{  zIndex :'999999999999'}} />
```

Comment:
Why did we add this ?

Reply:
in register and verify register components i gave the main div position static for many purposes, unfortunately this mess with toast container and make it behind the div , if i make many requests the fist toast will not be affected but others will be behind my container

Reply:
I will check that later.